src/os/bluestore/BlueStore.cc

```diff
@@ -7493,6 +7501,60 @@ int BlueStore::_fsck(bool deep, bool repair)
 	} else {
 	  m.insert(o->onode.nid);
 	}
+	if (per_pool_omap &&
+	    (!o->onode.is_perpool_omap() && !o->onode.is_pgmeta_omap())) {
+	  derr << "fsck error: " << oid << " has omap that is not per-pool or pgmeta"
+	       << dendl;
+	  ++errors;
```

Comment:
This will result in a mismatch in error count and repaired count if any. Due to the fact that repairer isn't involved in the repair below